src/Umbraco.Core/Persistence/Repositories/Implement/ContentTypeRepositoryBase.cs

```diff
@@ -521,9 +521,12 @@ protected void ValidateAlias(PropertyType pt)
         {
             if (string.IsNullOrWhiteSpace(pt.Alias))
             {
-                var m = $"Property Type '{pt.Name}' cannot have an empty Alias. This is most likely due to invalid characters stripped from the Alias.";
-                var e = new InvalidOperationException(m);
-                Logger.Error<ContentTypeRepositoryBase<TEntity>>(m, e);
+                var e = new InvalidOperationException($"Property Type '{pt.Name}' cannot have an empty Alias. This is most likely due to invalid characters stripped from the Alias.");
```

Comment:
Another case where we could have an extension method to just pass in a message ? this ones is a little more tricky with the typed exception and the message formatting, but might work somehow?